src/main/java/com/commercetools/sync/commons/helpers/BaseSyncStatistics.java

```diff
@@ -0,0 +1,237 @@
+package com.commercetools.sync.commons.helpers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.netty.util.internal.StringUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+import java.util.concurrent.TimeUnit;
+
+import static java.lang.String.format;
+
+public class BaseSyncStatistics {
```

Comment:
I wonder if class with such summary shouldn't be immutable? And has some additional builder for use in Syncers. But it's just a thought, probably needless.